susemanager-utils/susemanager-sls/salt/util/mgr_switch_to_venv_minion.sls

```diff
@@ -111,3 +111,12 @@ mgr_salt_minion_of_another_master:
   test.fail_without_changes:
     - comment: The salt-minion is managed by another master
 {%- endif %}
+{%- if grains['transactional'] == True -%}
+mgr_enable_venv_salt-minion:
+  cmd.run:
+    - name: systemctl enable venv-salt-minion
+{%- else -%}
+mgr_enable_venv_salt-minion:
+  service.running:
+    - name: venv-salt-minion.service
```

Comment:
This implementation (with additional arguments) is already defined in lines 59-64. I would drop this part and move the Jinja2 if/else control structure to the existing definition (together with the transactional-update branch.